### ISSUE TITLE
ARAX returns 0 results from COHD while the COHD endpoint does return results

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#1579]
Contrast: https://arax.ncats.io/?r=16508 that shows 55 results from only ICEES while https://arax.ncats.io/?source=ARS&id=74185c06-0bc8-4189-8291-f87fa0d4db85 shows 500 results from COHD. It appears that with this query:
```
{
  "edges": {
    "e01": {
      "object": "n1",
      "predicates": [
        "biolink:correlated_with"
      ],
      "subject": "n0"
    }
  },
  "nodes": {
    "n0": {
      "categories": [
        "biolink:Disease"
      ],
      "ids": [
        "MONDO:0004979"
      ],
      "is_set": false
    },
    "n1": {
      "categories": [
        "biolink:ChemicalSubstance"
      ],
      "is_set": false
    }
  }
}
```
we are not able to get results from COHD successfully. Could be an identifier issue (see https://github.com/NCATSTranslator/testing/issues/18#issuecomment-882813346 )

**Comment by chunyuma** [comment]
I think this issue can only be solved by calling COHD API. In ARAX, we have removed SNOMED CT from KG2 so `MONDO:0004979` doesn't have synonym `SNOMED:195967001`.

**Comment by chunyuma** [comment]
@amykglen, is it easy to integrate COHD API into `expand`? So the case like this happens, we can call COHD API directly and use their results.

**Comment by amykglen** [comment]
yes - definitely. how do you want to communicate when the API should be used? would it be when `slow_mode=True`?

if you add the new parameter that captures that info and let me know, I can hook it up so that the API is used in that case.

**Comment by chunyuma** [comment]
Thanks @amykglen. Yes, we want to call API when `slow_mode=True` I didn't add a new parameter to capture that info. My thought is to manually set `slow_mode=True` when we need it. If we set `slow_mode=True`, it should switch to call API instead of using `COHD_qurier.py`. Is it possible?

**Comment by amykglen** [comment]
yep, that should work fine. so you can just add a `slow_mode` parameter to the [COHD command definition section](https://github.com/RTXteam/RTX/blob/master/code/ARAX/ARAXQuery/Expand/expand_utilities.py#L622-L655) with `"default": "false"` and then I can take care of the rest.

**Comment by chunyuma** [comment]
@amykglen, I've added a `COHD_slow_mode` parameter to the COHD command definition section. Thanks for your help!

**Comment by amykglen** [comment]
alright, it's all plugged in in `master`, @chunyuma!

**Comment by amykglen** [comment]
I also regenerated `DSL_Documentation.md` so that it includes this new parameter

**Comment by chunyuma** [comment]
Thanks @amykglen! I really appreciate your help.

**Comment by amykglen** [comment]
no problem. and just fyi, I believe @isbluis will have to regenerate some UI stuff before the new parameter will appear as an option in the DSL builder dropdown.

**Comment by isbluis** [comment]
Hi @amykglen .  I think the UI already has this as of 8049948 this morning, and should be live.
But please let me know if this is not the case and/or something is amiss.
Thanks!

**Comment by amykglen** [comment]
ah, whoops, didn't realize it had made it in the rollout! sorry. thanks!

**Comment by isbluis** [comment]
No worries at all!  Lots of weekend commit activity!

**Comment by chunyuma** [comment]
This issue should also have been solved via the method used in #1625. I checked this query by using the new method and it works locally. @dkoslicki, please help me check it on arax.ncats.io. If it still works, then I think we can close this issue. Thanks!

**Comment by dkoslicki** [comment]
Fewer than the original 500 COHD results are being returned, but now at least we are getting some results from COHD. Here’s the results for the TRAPI 1.2 query: https://arax.ncats.io/?r=24407. I think this should be good to close
